src/components/LoginForm.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 
-const LoginForm = ({ Login, error, handleClose }) => {
+const LoginForm = ({ Login, error, SignUp, handleClose }) => {
```

Comment:
Not really an important detail, but is there a reason these function are capitalized while the others are not?

Reply:
None - i'm going to have to go back in and modify some things once the routes are set up. Good catch though